.github/workflows/release_python.yml

```
@@ -84,6 +84,16 @@ jobs:
           args: --release -o dist -i python3.10 --features=pyo3/extension-module,services-all
           sccache: true
           manylinux: auto
+      - name: Build free-threaded wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          maturin-version: "1.7.7"
```

```suggestion
          maturin-version: "1.7.7"
```

this is not required.